sdcm/logcollector.py

```diff
@@ -469,14 +469,29 @@ class GrafanaSnapshot(GrafanaEntity):
     Extends:
         GrafanaEntity
     """
-
+    panels_load_timeout = 10
+    scroll_step = 1000
     snapshot_locators_sequence = [
         (By.XPATH, """//button[contains(@class, "navbar-button--share")]"""),
         (By.XPATH, """//ul/li[contains(text(), "Snapshot")]"""),
         (By.XPATH, """//button//span[contains(text(), "Publish to snapshot.raintank.io")]"""),
         (By.XPATH, """//a[contains(@href, "https://snapshot.raintank.io")]""")
     ]
 
+    def scrolldown_dashboards_view(self, remote_browser):
+        scroll_ready_element = (By.XPATH, """//span[contains(text(), "Total Nodes")]""")
+        scrolling_element = (By.XPATH, "//div[@class='view']")
```

Comment:
Do you find it a better to move constants to class level?

Reply:
moved